_posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Google Cloud Loader 0.2.0 released"
+title-short: Google Cloud Loader 0.2.0
+tags: [snowplow, gcs, gcp, loader ]
+author: Ben
+category: Releases
+permalink: /blog/2019/12/11/gcs-loader-0.2.0-released/
+---
+
+We are pleased to announce [version 0.2.0][release-020] of Google Cloud Storage loader.
+This small release is centered around critical bug fixes and updated default configuration.
+
+<!--more-->
+
+<h2 id="shards">1. Default shard number</h2>
+
+Google Dataflow is Google's job runner implementation that allows for "zero-configuration" scaling.
+The autoscaling enables dynamic worker number adjustment to the needs of a pipeline.
+Dataflow automatically detects imbalances in work assignments and can dynamically reassign underused workers to decrease job's overall processing time. However, setting a fixed number of shards (maintained throughout the job run) can negatively impact the dynamic work reassignment by either scattering data too much or throttling writes. Therefore making runner control the number of shards makes a better default and should comply to any environment that it is being run.
+
+To preserve the old behaviour set `--numShards=1` when running the job.
+
+<h2 id="roadmap">2. Roadmap</h2>
```

Comment:
Do we need an upgrading section? Per https://snowplowanalytics.com/blog/2019/10/11/iglu-server-0.6.0-released/